Alignment/ReferenceTrajectories/src/GblTrajectory.cc

```diff
@@ -1077,13 +1079,15 @@ namespace gbl {
             if (measDim > 2) {
               matPDer = matP * matDer;
             } else { // 'shortcut' for position measurements
+	      matPDer.setZero();
```

Comment:
@mschrode please stick to the indentation. This is likely due to tabs in the preceding whitespace.